### DISCUSSION TITLE
V2 피드백

### DISCUSSION THREAD

**Discussion by wonderlandpark** [koreanbots/koreanbots#392] [일반] [answered]
# 한국 디스코드봇 리스트가 새로운 모습으로 여러분께 인사드립니다!

안녕하세요, 한국 디스코드봇 리스트입니다.

한국 디스코드봇 리스트가 새로운 버전인 V2로 변경되었습니다.
따라서, 여러가지 변경 사항이 있습니다!

## 디자인 변경
전체적인 디자인이 변경되었습니다!

## 하트 방식 변경
- 기존: 최초 1회만 하트를 누를 수 있는 방식
- 변경: 12시간마다 하트를 누를 수 있는 방식

자세한 내용은 해당 글을 참고해주세요.
https://blog.koreanbots.dev/koreanbots-v2-changes/

지금 바로 새로운 모습을 확인하세요
**__[koreanbots.dev](https://koreanbots.dev)__**

<hr /><em>This discussion was created from the release <a href='https://github.com/koreanbots/koreanbots/releases/tag/v2.0.0'>V2 Released</a>.</em>

**Comment by aeongdesu** [comment]
https://github.com/koreanbots/koreanbots/discussions/391
지워졌다고 뜹니다, 수정 해주세요~

**Comment by wonderlandpark** [comment, marked as answer]
피드백 감사드립니다. 다른 의견은 새로운 Discussion 을 생성해주세요
